apps/meteor/server/modules/watchers/lib/messages.ts

```diff
@@ -13,12 +13,26 @@ const getUserNameCached = mem(
 	{ maxAge: 10000 },
 );
 
+const isMutedUnmuted = (messageType: string): boolean => {
```

Comment:
I see there's a `getHiddenSystemMessages` function with some similar logic. I wonder if we could unify both 🤔 

Like creating a `lib` that did this `mute_unmute` parsing on a single place (without importing the `settings.watch` from there)

wdyt?